auth-web/src/util/common-util.ts

```diff
@@ -148,6 +148,11 @@ export default class CommonUtils {
   static formatDatePickerDate (date?: Date) {
     return moment(date || new Date()).format('YYYY-MM-DD')
   }
+
+  static formatDateToHumanReadable (date: Date | string | moment.Moment) {
```

Comment:
couldn't we use the function below? it's the same thing except for the new Date(), but we can just set that as the default?

```
 static formatCurrentDate (date = new Date()) {
    return moment(date).format('MMMM DD, YYYY')
  }
```